staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/managedfields_test.go

```diff
@@ -462,6 +483,17 @@ func TestSortEncodedManagedFields(t *testing.T) {
 				{Manager: "c", Operation: metav1.ManagedFieldsOperationUpdate, Time: &metav1.Time{time.Date(2000, time.January, 0, 0, 0, 0, 1, time.UTC)}},
 			},
 		},
+		{
+			name: "entries with subresource field",
+			managedFields: []metav1.ManagedFieldsEntry{
+				{Manager: "a", Operation: metav1.ManagedFieldsOperationApply, Subresource: "scale"},
```

Comment:
maybe add in a second subresource to prove that those are sorted correctly?

Reply:
Makes sense 👍